tools/tutorial_fleet_management/fleet-connector/fleet-connector.py

```diff
@@ -0,0 +1,73 @@
+from ankaios_sdk import Workload, Ankaios, WorkloadStateEnum, WorkloadSubStateEnum, AnkaiosLogLevel, Manifest, Request, CompleteState
+import paho.mqtt.client as mqtt
+import json
+import os
+import logging
+import sys
+
+logger = logging.getLogger("fleetconnector")
+stdout = logging.StreamHandler(stream=sys.stdout)
+stdout.setLevel(logging.INFO)
+logger.addHandler(stdout)
+logger.setLevel(logging.INFO)
+
+# Configuration for MQTT  broker and topics
+BROKER = os.environ.get('MQTT_BROKER_ADDR', 'localhost')
+PORT = int(os.environ.get('MQTT_BROKER_PORT', '1883'))
+VEHICLE_ID = os.environ.get('VIN')
+BASE_TOPIC = f"vehicle/{VEHICLE_ID}"
+
+# Create a new Ankaios object.
+# The connection to the control interface is automatically done at this step.
+ankaios = Ankaios()
+
+# Callback when the client receives a CONNACK response from the MQTT server
+def on_connect(client, userdata, flags, reason_code, properties):
+    client.subscribe(f"{BASE_TOPIC}/manifest/apply/req")
+    client.subscribe(f"{BASE_TOPIC}/manifest/delete/req")
+    client.subscribe(f"{BASE_TOPIC}/state/req")
+
+def convert_manifest_result_to_dict(result):
+    dict = { "added_workloads": [], "deleted_workloads": [] }
+    for a in result["added_workloads"]:
+        dict["added_workloads"].append(a.__dict__)
+    for d in result["deleted_workloads"]:
+        dict["deleted_workloads"].append(d.__dict__)
+    return dict
+
```

Comment:
We don't really need this conversion function as the SDK is already providing a dict:
```suggestion
```